app/models/miq_provision_orch_workflow.rb

```diff
@@ -0,0 +1,67 @@
+class MiqProvisionOrchWorkflow < MiqProvisionVirtWorkflow
+  def initialize(values, requester, options = {})
```

Comment:
This method is copied from `MiqProvisionVirtWorkflow`. Only changed the class name from `VmOrTemplate` to 'OrchestrationTemplate`.